cliconfig/context.go

```diff
@@ -86,6 +87,20 @@ func (c *Config) AddContext(context *Context) error {
 	return nil
 }
 
+// RemoveContext remove a context from the list of known contexts
+func (c *Config) RemoveContext(name string) error {
+	if name == "localhost" {
+		return ErrRemoveDefaultContext
+	}
```

Comment:
Why block the remove of the default context ? For me every context can be deleted. It does not make sense to block just this one.

Also, please ensure that there is no error if a user removes all the contexts. In that case, the default context should be used.